src/main/io/gps.c

```diff
@@ -1655,7 +2194,7 @@ static bool UBLOX_parse_gps(void)
         }
 #endif
         break;
-    case MSG_SVINFO:
+    case CLSMSG(CLASS_NAV, MSG_NAV_SVINFO):
```

Comment:
Adding a guard here for manually configured GPS modules would save a fair bit of unnecessary processing:
```
            if (ARMING_FLAG(ARMED)) {
                break;
            }
```

Reply:
Thanks - could you explain more clearly the code you'd suggest?  
There won't be packets of this type to parse unless connected to configurator.